.github/workflows/pull-request.yml

```diff
@@ -33,7 +33,14 @@ jobs:
         run: dotnet build Letterbook.sln --configuration Release
       
       - name: Test
-        run: dotnet test Letterbook.sln --configuration Release --logger GitHubActions
+        run: dotnet test Letterbook.sln --configuration Release --logger GitHubActions --blame-crash --no-build
```

Comment:
I think so, yes. I want the compiler to not skip any optimizations it would have otherwise done, and I want library code not to be doing anything clever that might change its behavior.